src/macaron/output_reporter/templates/macaron.html

```diff
@@ -164,6 +164,30 @@
 {% endcall %}
 {% endmacro %}
 
+{% macro render_run_checks(check_tree, excluded_checks) %}
+<ul class="tree-view-nested-list">
+    {% for key, first_level in check_tree.items() %}
```

Comment:
The first level of the tree (i.e checks that don't have a parent) don't need to have the symbol └── before it
![image](https://github.com/oracle-samples/macaron/assets/33509400/215c8193-4317-4a45-89ea-e5dd251944a5)

Therefore, we handle it separately.